react-instantsearch-hooks/range-slider-with-radix-ui/src/App.js

```diff
@@ -0,0 +1,84 @@
+import { useEffect, useState } from 'react';
+import {
+  Hits,
+  InstantSearch,
+  Pagination,
+  RefinementList,
+  SearchBox,
+  useConnector,
+} from 'react-instantsearch-hooks-web';
+import connectRange from 'instantsearch.js/es/connectors/range/connectRange';
+import * as Slider from '@radix-ui/react-slider';
+import algoliasearch from 'algoliasearch/lite';
+
+import './App.css';
+
+const searchClient = algoliasearch(
+  'B1G2GM9NG0',
+  'aadef574be1f9252bb48d4ea09b5cfe5'
+);
+
+function App() {
+  return (
+    <div className="container">
+      <InstantSearch
+        searchClient={searchClient}
+        indexName="demo_ecommerce"
+        insights={true}
+      >
+        <div className="search-panel">
+          <div className="search-panel__filters">
+            <RefinementList attribute="brand" />
+            <RangeSlider attribute="price" />
+          </div>
+
+          <div className="search-panel__results">
+            <SearchBox className="searchbox" placeholder="Search" />
+            <Hits />
+
+            <div className="pagination">
+              <Pagination />
+            </div>
+          </div>
+        </div>
+      </InstantSearch>
+    </div>
+  );
+}
+
+function useRangeSlider(props) {
```

Comment:
this should be useRange IMO (which we have in the lib already, no need to rewrite it)

Reply:
Ah, I didn't realize [the current docs were outdated](https://www.algolia.com/doc/api-reference/widgets/range-slider/react-hooks/). I'll change it.